scvi/module/_scanvae.py

```diff
@@ -275,28 +279,21 @@ def classification_loss(self, labelled_dataset):
 
     def loss(
         self,
-        tensors,
-        inference_outputs,
-        generative_ouputs,
-        feed_labels=False,
-        kl_weight=1,
-        labelled_tensors=None,
-        classification_ratio=None,
+        tensors: dict[str, torch.Tensor],
+        inference_outputs: dict[str, torch.Tensor | Distribution | None],
+        generative_ouputs: dict[str, Distribution | None],
+        kl_weight: float = 1.0,
+        labelled_tensors: dict[str, torch.Tensor] | None = None,
+        classification_ratio: float | None = None,
     ):
         """Compute the loss."""
         px = generative_ouputs["px"]
         qz1 = inference_outputs["qz"]
         z1 = inference_outputs["z"]
         x = tensors[REGISTRY_KEYS.X_KEY]
         batch_index = tensors[REGISTRY_KEYS.BATCH_KEY]
+        y = None
 
-        if feed_labels:
-            y = tensors[REGISTRY_KEYS.LABELS_KEY]
-        else:
-            y = None
-        is_labelled = False if y is None else True
-
-        # Enumerate choices of label
         ys, z1s = broadcast_labels(y, z1, n_broadcast=self.n_labels)
```

Comment:
could probably clean up this function call since `y` is always `None` - it would help with debugging and addressing #2429